.github/workflows/tests.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 env:
   CACHE_NUMBER: 0  # increase to reset cache manually
-  PYTEST_FLAGS: --tardis-refdata=${{ github.workspace }}/tardis-refdata --tardis-regression-data=${{ github.workspace }}/tardis-regression-data
+  PYTEST_FLAGS: --tardis-refdata=${{ github.workspace }}/tardis-refdata --tardis-regression-data=${{ github.workspace }}/tardis-regression-data -vvv
```

Comment:
Should `-vvv` be removed now?

Reply:
I was thinking to keep it, it helps tell us exactly what tests are skipped right?

Reply:
I agree, but it also makes the logs very long. I think it should be removed unless we are doing specific debugging.

Reply:
Done in 9dd7c854ffa842e3e40f6317bc7533269f33a97e